pages/02.Documentation/02.compute/docs.de.md

```diff
@@ -31,14 +31,13 @@ M1 Medium   |  m1.medium  | 16GB   |   4   |   50GB   |
 M1 Large    |  m1.large   | 32GB   |   8   |   50GB   |
 
 (*)
-You can extend storage using our our Block Storage Service.
-
-### Local SSD storage instance types (L1)
+Sie können den Root-Speicher durch unseren ebenfalls verteilten Block-Speicher ergänzen.
 
-Local SSD storage instances offer low latency SSD storage directly on the local host.
-These can be useful for special workloads like replicated databases.
+### Local SSD Storage Instanz-Typen (L1)
 
-! Availability and data durability are reduced, because data is only stored locally on one server.
+Local SSD Storage Instanz-Typen sind mit einem lokalen Root-Speichermedium in unterschiedlichen Größen ausgestattet.
```

Comment:
Irgendwie gefällt mir der Informationsgehalt des englischen ersten satzes besser, der den low-latency storage hervorhebt

Reply:
The diff seems to be confused by whitespace and the lines do not match exactly. What you are missing seems to be in the next line(s).